[2016-08-09 08:53:18,462] INFO  [main] play.Logger.info(Logger.java:308) - Starting C:\Users\Administrator.SC-201606081350\Desktop\wugake\wgk
[2016-08-09 08:53:19,509] WARN  [main] play.Logger.warn(Logger.java:367) - You're running Play! in DEV mode
[2016-08-09 08:53:19,604] INFO  [main] play.Logger.info(Logger.java:308) - Listening for HTTP on port 80 (Waiting a first request to start) ...
[2016-08-09 08:56:25,010] INFO  [MLog-Init-Reporter] com.mchange.v2.log.MLog$1.run(MLog.java:124) - MLog clients using log4j logging.
[2016-08-09 08:56:25,171] INFO  [play-thread-1] com.mchange.v2.c3p0.C3P0Registry.banner(C3P0Registry.java:248) - Initializing c3p0-0.9.5 [built 02-January-2015 13:25:04 -0500; debug? true; trace: 10]
[2016-08-09 08:56:25,251] INFO  [play-thread-1] com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getPoolManager(AbstractPoolBackedDataSource.java:522) - Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 10, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 1000, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hge10a9iu6s0dh1m5dnfu|729e4f7c, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceUseNamedDriverClass -> false, identityToken -> 1hge10a9iu6s0dh1m5dnfu|729e4f7c, idleConnectionTestPeriod -> 10, initialPoolSize -> 3, jdbcUrl -> jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 30, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 10, numHelperThreads -> 3, preferredTestQuery -> /* ping */ SELECT 1, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> true, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
[2016-08-09 08:56:25,299] WARN  [play-thread-1] com.mchange.v2.resourcepool.BasicResourcePool.<init>(BasicResourcePool.java:288) - Bad pool size config, start 3 < min 10. Using 10 as start.
[2016-08-09 08:56:25,340] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Connected to jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8 for default
[2016-08-09 08:56:26,355] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Application 'wgk' is now started !
[2016-08-09 08:56:26,685] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 08:56:43,668] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@712m83gap
buyer.tasks2 action not found

Action not found
Action buyer.tasks2 could not be found. Error raised is Controller controllers.buyer not found

play.exceptions.ActionNotFoundException: Action buyer.tasks2 not found
	at play.mvc.ActionInvoker.getActionMethod(ActionInvoker.java:566)
	at play.mvc.ActionInvoker.resolve(ActionInvoker.java:84)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.Exception: Controller controllers.buyer not found
	... 3 more
[2016-08-09 08:56:50,824] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 08:59:17,908] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@712m83gb0
buyer.tasks2 action not found

Action not found
Action buyer.tasks2 could not be found. Error raised is Controller controllers.buyer not found

play.exceptions.ActionNotFoundException: Action buyer.tasks2 not found
	at play.mvc.ActionInvoker.getActionMethod(ActionInvoker.java:566)
	at play.mvc.ActionInvoker.resolve(ActionInvoker.java:84)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.Exception: Controller controllers.buyer not found
	... 3 more
[2016-08-09 09:03:42,304] INFO  [main] play.Logger.info(Logger.java:308) - Starting C:\Users\Administrator.SC-201606081350\Desktop\wugake\wgk
[2016-08-09 09:03:43,281] WARN  [main] play.Logger.warn(Logger.java:367) - You're running Play! in DEV mode
[2016-08-09 09:03:43,392] INFO  [main] play.Logger.info(Logger.java:308) - Listening for HTTP on port 80 (Waiting a first request to start) ...
[2016-08-09 09:03:53,383] INFO  [MLog-Init-Reporter] com.mchange.v2.log.MLog$1.run(MLog.java:124) - MLog clients using log4j logging.
[2016-08-09 09:03:53,454] INFO  [play-thread-1] com.mchange.v2.c3p0.C3P0Registry.banner(C3P0Registry.java:248) - Initializing c3p0-0.9.5 [built 02-January-2015 13:25:04 -0500; debug? true; trace: 10]
[2016-08-09 09:03:53,524] INFO  [play-thread-1] com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getPoolManager(AbstractPoolBackedDataSource.java:522) - Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 10, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 1000, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hge10a9iu71mbqnolsck|6a39d072, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceUseNamedDriverClass -> false, identityToken -> 1hge10a9iu71mbqnolsck|6a39d072, idleConnectionTestPeriod -> 10, initialPoolSize -> 3, jdbcUrl -> jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 30, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 10, numHelperThreads -> 3, preferredTestQuery -> /* ping */ SELECT 1, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> true, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
[2016-08-09 09:03:53,534] WARN  [play-thread-1] com.mchange.v2.resourcepool.BasicResourcePool.<init>(BasicResourcePool.java:288) - Bad pool size config, start 3 < min 10. Using 10 as start.
[2016-08-09 09:03:53,574] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Connected to jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8 for default
[2016-08-09 09:03:54,184] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Application 'wgk' is now started !
[2016-08-09 09:03:54,401] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 09:07:30,630] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@712m9ebc2
buyer.tasks2 action not found

Action not found
Action buyer.tasks2 could not be found. Error raised is Controller controllers.buyer not found

play.exceptions.ActionNotFoundException: Action buyer.tasks2 not found
	at play.mvc.ActionInvoker.getActionMethod(ActionInvoker.java:566)
	at play.mvc.ActionInvoker.resolve(ActionInvoker.java:84)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.Exception: Controller controllers.buyer not found
	... 3 more
[2016-08-09 09:08:36,914] INFO  [main] play.Logger.info(Logger.java:308) - Starting C:\Users\Administrator.SC-201606081350\Desktop\wugake\wgk
[2016-08-09 09:08:38,635] WARN  [main] play.Logger.warn(Logger.java:367) - You're running Play! in DEV mode
[2016-08-09 09:08:39,248] INFO  [main] play.Logger.info(Logger.java:308) - Listening for HTTP on port 80 (Waiting a first request to start) ...
[2016-08-09 09:09:20,525] INFO  [MLog-Init-Reporter] com.mchange.v2.log.MLog$1.run(MLog.java:124) - MLog clients using log4j logging.
[2016-08-09 09:09:20,677] INFO  [play-thread-1] com.mchange.v2.c3p0.C3P0Registry.banner(C3P0Registry.java:248) - Initializing c3p0-0.9.5 [built 02-January-2015 13:25:04 -0500; debug? true; trace: 10]
[2016-08-09 09:09:20,777] INFO  [play-thread-1] com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getPoolManager(AbstractPoolBackedDataSource.java:522) - Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 10, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 1000, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hge10a9iu78mrbqw8bii|141e00a, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceUseNamedDriverClass -> false, identityToken -> 1hge10a9iu78mrbqw8bii|141e00a, idleConnectionTestPeriod -> 10, initialPoolSize -> 3, jdbcUrl -> jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 30, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 10, numHelperThreads -> 3, preferredTestQuery -> /* ping */ SELECT 1, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> true, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
[2016-08-09 09:09:20,812] WARN  [play-thread-1] com.mchange.v2.resourcepool.BasicResourcePool.<init>(BasicResourcePool.java:288) - Bad pool size config, start 3 < min 10. Using 10 as start.
[2016-08-09 09:09:20,869] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Connected to jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8 for default
[2016-08-09 09:09:21,953] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Application 'wgk' is now started !
[2016-08-09 09:09:22,193] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 09:14:27,058] INFO  [jobs-thread-3] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 09:17:30,016] ERROR [jobs-thread-2] play.Logger.niceThrowable(Logger.java:604) - 

@712mame3b
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/controllers/UserCenter.java around line 903)
The file /app/controllers/UserCenter.java could not be compiled. Error raised is : Duplicate method buyerAccount() in type UserCenter

play.exceptions.CompilationException: Duplicate method buyerAccount() in type UserCenter
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:480)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-08-09 09:20:11,254] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 09:20:18,321] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 09:20:20,587] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 09:20:22,627] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 09:23:12,783] ERROR [jobs-thread-8] play.Logger.niceThrowable(Logger.java:604) - 

@712mame3d
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/controllers/UserCenter.java around line 991)
The file /app/controllers/UserCenter.java could not be compiled. Error raised is : BuyerAccount2 cannot be resolved to a type

play.exceptions.CompilationException: BuyerAccount2 cannot be resolved to a type
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-08-09 09:26:24,212] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 09:26:34,389] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 09:26:37,599] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 09:26:40,022] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 09:26:41,705] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 09:26:43,583] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 09:27:06,831] INFO  [play-thread-1] controllers.UserCenter.checkBeforeSaveBuyerAccount2(UserCenter.java:1047) - Buyer=1678-maimai save buyerAccount: Check Duplicate
[2016-08-09 09:27:06,851] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 09:27:06,851] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - 
### Error updating database.  Cause: java.sql.SQLException: Data truncated for column 'platform' at row 1
### The error may involve models.mappers.BuyerAccountMapper2.insert-Inline
### The error occurred while setting parameters
### SQL: insert into buyer_account(nick,platform,consignee,state,city,region,address,mobile,user_id,status,create_time,modify_time) values(?,?,?,?,?,?,?,?,?,?,?,?)
### Cause: java.sql.SQLException: Data truncated for column 'platform' at row 1
org.apache.ibatis.exceptions.PersistenceException: 
### Error updating database.  Cause: java.sql.SQLException: Data truncated for column 'platform' at row 1
### The error may involve models.mappers.BuyerAccountMapper2.insert-Inline
### The error occurred while setting parameters
### SQL: insert into buyer_account(nick,platform,consignee,state,city,region,address,mobile,user_id,status,create_time,modify_time) values(?,?,?,?,?,?,?,?,?,?,?,?)
### Cause: java.sql.SQLException: Data truncated for column 'platform' at row 1
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:26)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:154)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:141)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:51)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:52)
	at com.sun.proxy.$Proxy70.insert(Unknown Source)
	at models.BuyerAccount2.save(BuyerAccount2.java:236)
	at controllers.UserCenter.saveBuyerAccount2(UserCenter.java:1025)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
Caused by: java.sql.SQLException: Data truncated for column 'platform' at row 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:998)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3835)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3771)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2435)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2582)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2535)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1911)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1203)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.execute(NewProxyPreparedStatement.java:1392)
	at sun.reflect.GeneratedMethodAccessor26.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.ibatis.logging.jdbc.PreparedStatementLogger.invoke(PreparedStatementLogger.java:62)
	at com.sun.proxy.$Proxy20.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:44)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:69)
	at org.apache.ibatis.executor.ReuseExecutor.doUpdate(ReuseExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:105)
	at org.apache.ibatis.executor.CachingExecutor.update(CachingExecutor.java:71)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:152)
	... 32 more
[2016-08-09 09:31:33,318] INFO  [main] play.Logger.info(Logger.java:308) - Starting C:\Users\Administrator.SC-201606081350\Desktop\wugake\wgk
[2016-08-09 09:31:34,212] WARN  [main] play.Logger.warn(Logger.java:367) - You're running Play! in DEV mode
[2016-08-09 09:31:34,323] INFO  [main] play.Logger.info(Logger.java:308) - Listening for HTTP on port 80 (Waiting a first request to start) ...
[2016-08-09 09:31:44,781] INFO  [MLog-Init-Reporter] com.mchange.v2.log.MLog$1.run(MLog.java:124) - MLog clients using log4j logging.
[2016-08-09 09:31:44,852] INFO  [play-thread-1] com.mchange.v2.c3p0.C3P0Registry.banner(C3P0Registry.java:248) - Initializing c3p0-0.9.5 [built 02-January-2015 13:25:04 -0500; debug? true; trace: 10]
[2016-08-09 09:31:44,922] INFO  [play-thread-1] com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getPoolManager(AbstractPoolBackedDataSource.java:522) - Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 10, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 1000, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hge10a9iu81fzgyivtgl|1d87b360, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceUseNamedDriverClass -> false, identityToken -> 1hge10a9iu81fzgyivtgl|1d87b360, idleConnectionTestPeriod -> 10, initialPoolSize -> 3, jdbcUrl -> jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 30, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 10, numHelperThreads -> 3, preferredTestQuery -> /* ping */ SELECT 1, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> true, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
[2016-08-09 09:31:44,952] WARN  [play-thread-1] com.mchange.v2.resourcepool.BasicResourcePool.<init>(BasicResourcePool.java:288) - Bad pool size config, start 3 < min 10. Using 10 as start.
[2016-08-09 09:31:44,992] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Connected to jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8 for default
[2016-08-09 09:31:45,602] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Application 'wgk' is now started !
[2016-08-09 09:31:45,812] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 09:32:18,116] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [platform] Required
[2016-08-09 09:33:02,858] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [platform] Required
[2016-08-09 09:33:50,602] INFO  [main] play.Logger.info(Logger.java:308) - Starting C:\Users\Administrator.SC-201606081350\Desktop\wugake\wgk
[2016-08-09 09:33:51,504] WARN  [main] play.Logger.warn(Logger.java:367) - You're running Play! in DEV mode
[2016-08-09 09:33:51,575] INFO  [main] play.Logger.info(Logger.java:308) - Listening for HTTP on port 80 (Waiting a first request to start) ...
[2016-08-09 09:34:16,498] INFO  [MLog-Init-Reporter] com.mchange.v2.log.MLog$1.run(MLog.java:124) - MLog clients using log4j logging.
[2016-08-09 09:34:16,579] INFO  [play-thread-1] com.mchange.v2.c3p0.C3P0Registry.banner(C3P0Registry.java:248) - Initializing c3p0-0.9.5 [built 02-January-2015 13:25:04 -0500; debug? true; trace: 10]
[2016-08-09 09:34:16,639] INFO  [play-thread-1] com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getPoolManager(AbstractPoolBackedDataSource.java:522) - Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 10, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 1000, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hge10a9iu84p1tlpmhb9|10469e8, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceUseNamedDriverClass -> false, identityToken -> 1hge10a9iu84p1tlpmhb9|10469e8, idleConnectionTestPeriod -> 10, initialPoolSize -> 3, jdbcUrl -> jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 30, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 10, numHelperThreads -> 3, preferredTestQuery -> /* ping */ SELECT 1, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> true, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
[2016-08-09 09:34:16,659] WARN  [play-thread-1] com.mchange.v2.resourcepool.BasicResourcePool.<init>(BasicResourcePool.java:288) - Bad pool size config, start 3 < min 10. Using 10 as start.
[2016-08-09 09:34:16,719] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Connected to jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8 for default
[2016-08-09 09:34:17,425] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Application 'wgk' is now started !
[2016-08-09 09:34:17,615] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 09:34:25,610] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 09:34:43,309] INFO  [play-thread-1] controllers.UserCenter.checkBeforeSaveBuyerAccount2(UserCenter.java:1047) - Buyer=1678-maimai save buyerAccount: Check Duplicate
[2016-08-09 09:34:44,917] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 09:35:42,922] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 09:35:45,459] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 09:39:19,282] ERROR [jobs-thread-3] play.Logger.niceThrowable(Logger.java:604) - 

@712mdj0nl
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/controllers/UserCenter.java around line 1166)
The file /app/controllers/UserCenter.java could not be compiled. Error raised is : The method platform(Platform) in the type BuyerAccountSearchVo is not applicable for the arguments (Platform2)

play.exceptions.CompilationException: The method platform(Platform) in the type BuyerAccountSearchVo is not applicable for the arguments (Platform2)
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-08-09 09:40:05,191] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 09:40:08,006] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 09:40:14,754] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 09:41:00,316] INFO  [play-thread-1] controllers.UserCenter.checkBeforeSaveBuyerAccount2(UserCenter.java:1047) - Buyer=1678-maimai save buyerAccount: Check Duplicate
[2016-08-09 09:41:08,950] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 09:45:39,669] INFO  [play-thread-1] controllers.UserCenter.checkBeforeSaveBuyerAccount2(UserCenter.java:1047) - Buyer=1678-maimai save buyerAccount: Check Duplicate
[2016-08-09 09:46:56,055] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 09:48:23,623] INFO  [play-thread-1] controllers.UserCenter.checkBeforeSaveBuyerAccount(UserCenter.java:1091) - Buyer=1678-maimai save buyerAccount: Check Duplicate
[2016-08-09 09:49:57,745] ERROR [jobs-thread-7] play.Logger.niceThrowable(Logger.java:604) - 

@712mdj0nn
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/controllers/UserCenter.java around line 1235)
The file /app/controllers/UserCenter.java could not be compiled. Error raised is : Duplicate method accountTaskStatus(BuyerTaskSearchVo) in type UserCenter

play.exceptions.CompilationException: Duplicate method accountTaskStatus(BuyerTaskSearchVo) in type UserCenter
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:480)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-08-09 09:50:44,659] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 09:51:01,240] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 09:52:33,892] INFO  [play-thread-1] controllers.UserCenter.checkBeforeSaveBuyerAccount2(UserCenter.java:1048) - Buyer=1678-maimai save buyerAccount: Check Duplicate
[2016-08-09 09:55:27,506] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 09:55:35,004] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 09:58:21,137] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 09:58:23,750] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 09:58:23,750] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - The validated string is empty
java.lang.IllegalArgumentException: The validated string is empty
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:321)
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:339)
	at controllers.TaskExecutor.isAvailableBuyerAccountAndItem(TaskExecutor.java:930)
	at controllers.TaskCenter.isTakeTasks2(TaskCenter.java:712)
	at controllers.TaskCenter.listTasks2(TaskCenter.java:402)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-08-09 09:58:46,166] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 10:00:44,696] INFO  [jobs-thread-1] jobs.AddKwdForFlow.doJob(AddKwdForFlow.java:38) - =====AddFlow Started=====
[2016-08-09 10:00:44,706] INFO  [jobs-thread-1] jobs.AddKwdForFlow.doJob(AddKwdForFlow.java:40) - 流量任务条数0
[2016-08-09 10:00:46,292] INFO  [jobs-thread-1] jobs.BuyerTaskCancel.doJob(BuyerTaskCancel.java:38) - cancel buyerTask start
[2016-08-09 10:00:46,304] INFO  [jobs-thread-1] jobs.BuyerTaskCancel.doJob(BuyerTaskCancel.java:49) - cancel buyerTask end
[2016-08-09 10:00:46,314] INFO  [jobs-thread-5] jobs.GetExpressNo.doJob(GetExpressNo.java:49) - =====GetExpressNo Started=====
[2016-08-09 10:00:47,058] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 10:00:52,788] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 10:00:52,788] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - The validated string is empty
java.lang.IllegalArgumentException: The validated string is empty
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:321)
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:339)
	at controllers.TaskExecutor.isAvailableBuyerAccountAndItem(TaskExecutor.java:930)
	at controllers.TaskCenter.isTakeTasks2(TaskCenter.java:712)
	at controllers.TaskCenter.listTasks2(TaskCenter.java:402)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-08-09 10:00:57,070] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 10:00:57,070] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - The validated string is empty
java.lang.IllegalArgumentException: The validated string is empty
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:321)
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:339)
	at controllers.TaskExecutor.isAvailableBuyerAccountAndItem(TaskExecutor.java:930)
	at controllers.TaskCenter.isTakeTasks2(TaskCenter.java:712)
	at controllers.TaskCenter.listTasks2(TaskCenter.java:402)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-08-09 10:02:24,155] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 10:02:24,155] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - The validated string is empty
java.lang.IllegalArgumentException: The validated string is empty
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:321)
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:339)
	at controllers.TaskExecutor.isAvailableBuyerAccountAndItem(TaskExecutor.java:930)
	at controllers.TaskCenter.isTakeTasks2(TaskCenter.java:712)
	at controllers.TaskCenter.listTasks2(TaskCenter.java:402)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-08-09 10:06:33,571] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 10:06:33,571] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - The validated string is empty
java.lang.IllegalArgumentException: The validated string is empty
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:321)
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:339)
	at controllers.TaskExecutor.isAvailableBuyerAccountAndItem(TaskExecutor.java:930)
	at controllers.TaskCenter.isTakeTasks2(TaskCenter.java:712)
	at controllers.TaskCenter.listTasks2(TaskCenter.java:402)
	at sun.reflect.GeneratedMethodAccessor684.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-08-09 10:06:36,044] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 10:06:36,044] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - The validated string is empty
java.lang.IllegalArgumentException: The validated string is empty
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:321)
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:339)
	at controllers.TaskExecutor.isAvailableBuyerAccountAndItem(TaskExecutor.java:930)
	at controllers.TaskCenter.isTakeTasks2(TaskCenter.java:712)
	at controllers.TaskCenter.listTasks2(TaskCenter.java:402)
	at sun.reflect.GeneratedMethodAccessor684.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-08-09 10:06:38,274] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 10:06:38,274] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - The validated string is empty
java.lang.IllegalArgumentException: The validated string is empty
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:321)
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:339)
	at controllers.TaskExecutor.isAvailableBuyerAccountAndItem(TaskExecutor.java:930)
	at controllers.TaskCenter.isTakeTasks2(TaskCenter.java:712)
	at controllers.TaskCenter.listTasks2(TaskCenter.java:402)
	at sun.reflect.GeneratedMethodAccessor684.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-08-09 10:08:23,153] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 10:08:23,153] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - The validated string is empty
java.lang.IllegalArgumentException: The validated string is empty
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:321)
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:339)
	at controllers.TaskExecutor.isAvailableBuyerAccountAndItem(TaskExecutor.java:930)
	at controllers.TaskCenter.isTakeTasks2(TaskCenter.java:712)
	at controllers.TaskCenter.listTasks2(TaskCenter.java:402)
	at sun.reflect.GeneratedMethodAccessor684.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-08-09 10:09:01,691] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 10:09:01,691] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - The validated string is empty
java.lang.IllegalArgumentException: The validated string is empty
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:321)
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:339)
	at controllers.TaskExecutor.isAvailableBuyerAccountAndItem(TaskExecutor.java:930)
	at controllers.TaskCenter.isTakeTasks2(TaskCenter.java:712)
	at controllers.TaskCenter.listTasks2(TaskCenter.java:402)
	at sun.reflect.GeneratedMethodAccessor684.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-08-09 10:09:48,481] INFO  [jobs-thread-6] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 10:11:04,034] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 10:11:04,034] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - The validated string is empty
java.lang.IllegalArgumentException: The validated string is empty
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:321)
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:339)
	at controllers.TaskExecutor.isAvailableBuyerAccountAndItem(TaskExecutor.java:930)
	at controllers.TaskCenter.isTakeTasks2(TaskCenter.java:710)
	at controllers.TaskCenter.listTasks2(TaskCenter.java:400)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-08-09 10:11:13,564] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 10:11:13,564] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - The validated string is empty
java.lang.IllegalArgumentException: The validated string is empty
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:321)
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:339)
	at controllers.TaskExecutor.isAvailableBuyerAccountAndItem(TaskExecutor.java:930)
	at controllers.TaskCenter.isTakeTasks2(TaskCenter.java:710)
	at controllers.TaskCenter.listTasks2(TaskCenter.java:400)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-08-09 10:11:46,448] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 10:11:46,448] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - The validated string is empty
java.lang.IllegalArgumentException: The validated string is empty
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:321)
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:339)
	at controllers.TaskExecutor.isAvailableBuyerAccountAndItem(TaskExecutor.java:930)
	at controllers.TaskCenter.isTakeTasks2(TaskCenter.java:710)
	at controllers.TaskCenter.listTasks2(TaskCenter.java:400)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-08-09 10:11:49,233] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 10:11:49,233] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - The validated string is empty
java.lang.IllegalArgumentException: The validated string is empty
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:321)
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:339)
	at controllers.TaskExecutor.isAvailableBuyerAccountAndItem(TaskExecutor.java:930)
	at controllers.TaskCenter.isTakeTasks2(TaskCenter.java:710)
	at controllers.TaskCenter.listTasks2(TaskCenter.java:400)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-08-09 10:11:51,743] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 10:11:51,743] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - The validated string is empty
java.lang.IllegalArgumentException: The validated string is empty
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:321)
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:339)
	at controllers.TaskExecutor.isAvailableBuyerAccountAndItem(TaskExecutor.java:930)
	at controllers.TaskCenter.isTakeTasks2(TaskCenter.java:710)
	at controllers.TaskCenter.listTasks2(TaskCenter.java:400)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-08-09 10:11:54,123] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 10:11:54,123] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - The validated string is empty
java.lang.IllegalArgumentException: The validated string is empty
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:321)
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:339)
	at controllers.TaskExecutor.isAvailableBuyerAccountAndItem(TaskExecutor.java:930)
	at controllers.TaskCenter.isTakeTasks2(TaskCenter.java:710)
	at controllers.TaskCenter.listTasks2(TaskCenter.java:400)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-08-09 10:19:48,523] INFO  [jobs-thread-2] jobs.AddKwdForFlow.doJob(AddKwdForFlow.java:38) - =====AddFlow Started=====
[2016-08-09 10:19:48,543] INFO  [jobs-thread-2] jobs.AddKwdForFlow.doJob(AddKwdForFlow.java:40) - 流量任务条数0
[2016-08-09 10:19:49,694] INFO  [jobs-thread-2] jobs.BuyerTaskCancel.doJob(BuyerTaskCancel.java:38) - cancel buyerTask start
[2016-08-09 10:19:49,704] INFO  [jobs-thread-2] jobs.BuyerTaskCancel.doJob(BuyerTaskCancel.java:49) - cancel buyerTask end
[2016-08-09 10:19:49,715] INFO  [jobs-thread-4] jobs.GetExpressNo.doJob(GetExpressNo.java:49) - =====GetExpressNo Started=====
[2016-08-09 10:21:05,807] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 10:21:05,817] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - The validated string is empty
java.lang.IllegalArgumentException: The validated string is empty
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:321)
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:339)
	at controllers.TaskExecutor.isAvailableBuyerAccountAndItem(TaskExecutor.java:930)
	at controllers.TaskCenter.isTakeTasks2(TaskCenter.java:709)
	at controllers.TaskCenter.listTasks2(TaskCenter.java:399)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-08-09 10:22:19,583] INFO  [main] play.Logger.info(Logger.java:308) - Starting C:\Users\Administrator.SC-201606081350\Desktop\wugake\wgk
[2016-08-09 10:22:20,573] WARN  [main] play.Logger.warn(Logger.java:367) - You're running Play! in DEV mode
[2016-08-09 10:22:20,684] INFO  [main] play.Logger.info(Logger.java:308) - Listening for HTTP on port 80 (Waiting a first request to start) ...
[2016-08-09 10:22:32,955] INFO  [MLog-Init-Reporter] com.mchange.v2.log.MLog$1.run(MLog.java:124) - MLog clients using log4j logging.
[2016-08-09 10:22:33,066] INFO  [play-thread-1] com.mchange.v2.c3p0.C3P0Registry.banner(C3P0Registry.java:248) - Initializing c3p0-0.9.5 [built 02-January-2015 13:25:04 -0500; debug? true; trace: 10]
[2016-08-09 10:22:33,146] INFO  [play-thread-1] com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getPoolManager(AbstractPoolBackedDataSource.java:522) - Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 10, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 1000, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hge10a9iu9uryy102mw4h|237aaeec, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceUseNamedDriverClass -> false, identityToken -> 1hge10a9iu9uryy102mw4h|237aaeec, idleConnectionTestPeriod -> 10, initialPoolSize -> 3, jdbcUrl -> jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 30, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 10, numHelperThreads -> 3, preferredTestQuery -> /* ping */ SELECT 1, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> true, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
[2016-08-09 10:22:33,166] WARN  [play-thread-1] com.mchange.v2.resourcepool.BasicResourcePool.<init>(BasicResourcePool.java:288) - Bad pool size config, start 3 < min 10. Using 10 as start.
[2016-08-09 10:22:33,206] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Connected to jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8 for default
[2016-08-09 10:22:33,913] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Application 'wgk' is now started !
[2016-08-09 10:22:34,120] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 10:22:49,745] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 10:22:49,745] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - The validated string is empty
java.lang.IllegalArgumentException: The validated string is empty
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:321)
	at org.apache.commons.lang.Validate.notEmpty(Validate.java:339)
	at controllers.TaskExecutor.isAvailableBuyerAccountAndItem(TaskExecutor.java:930)
	at controllers.TaskCenter.isTakeTasks2(TaskCenter.java:709)
	at controllers.TaskCenter.listTasks2(TaskCenter.java:399)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-08-09 10:23:24,372] INFO  [main] play.Logger.info(Logger.java:308) - Starting C:\Users\Administrator.SC-201606081350\Desktop\wugake\wgk
[2016-08-09 10:23:25,142] WARN  [main] play.Logger.warn(Logger.java:367) - You're running Play! in DEV mode
[2016-08-09 10:23:25,223] INFO  [main] play.Logger.info(Logger.java:308) - Listening for HTTP on port 80 (Waiting a first request to start) ...
[2016-08-09 10:26:44,969] INFO  [MLog-Init-Reporter] com.mchange.v2.log.MLog$1.run(MLog.java:124) - MLog clients using log4j logging.
[2016-08-09 10:26:45,070] INFO  [play-thread-1] com.mchange.v2.c3p0.C3P0Registry.banner(C3P0Registry.java:248) - Initializing c3p0-0.9.5 [built 02-January-2015 13:25:04 -0500; debug? true; trace: 10]
[2016-08-09 10:26:45,192] INFO  [play-thread-1] com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getPoolManager(AbstractPoolBackedDataSource.java:522) - Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 10, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 1000, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hge10a9iua06fm1nehumz|27cc7f4b, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceUseNamedDriverClass -> false, identityToken -> 1hge10a9iua06fm1nehumz|27cc7f4b, idleConnectionTestPeriod -> 10, initialPoolSize -> 3, jdbcUrl -> jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 30, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 10, numHelperThreads -> 3, preferredTestQuery -> /* ping */ SELECT 1, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> true, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
[2016-08-09 10:26:45,222] WARN  [play-thread-1] com.mchange.v2.resourcepool.BasicResourcePool.<init>(BasicResourcePool.java:288) - Bad pool size config, start 3 < min 10. Using 10 as start.
[2016-08-09 10:26:45,282] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Connected to jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8 for default
[2016-08-09 10:26:46,054] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Application 'wgk' is now started !
[2016-08-09 10:26:46,224] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 10:32:51,149] ERROR [play-thread-1] com.aton.base.secure.Secure.checkAuthentication(Secure.java:92) - Login admin not exists,admin_id=null
[2016-08-09 10:32:55,959] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [pass] Required
[2016-08-09 10:36:46,693] INFO  [jobs-thread-8] jobs.AddKwdForFlow.doJob(AddKwdForFlow.java:38) - =====AddFlow Started=====
[2016-08-09 10:36:46,701] INFO  [jobs-thread-8] jobs.AddKwdForFlow.doJob(AddKwdForFlow.java:40) - 流量任务条数0
[2016-08-09 10:36:49,111] INFO  [jobs-thread-8] jobs.BuyerTaskCancel.doJob(BuyerTaskCancel.java:38) - cancel buyerTask start
[2016-08-09 10:36:49,152] INFO  [jobs-thread-8] jobs.BuyerTaskCancel.doJob(BuyerTaskCancel.java:49) - cancel buyerTask end
[2016-08-09 10:36:49,172] INFO  [jobs-thread-7] jobs.GetExpressNo.doJob(GetExpressNo.java:49) - =====GetExpressNo Started=====
[2016-08-09 10:41:34,145] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-08-09 10:45:15,169] INFO  [play-thread-1] controllers.admins.UserManage.doRecharge(UserManage.java:108) - Mock Buying ingot=10000000 for user=1704
[2016-08-09 10:45:28,807] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 10:45:28,807] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - 
### Error updating database.  Cause: java.sql.SQLException: Data truncated for column 'log_type' at row 1
### The error may involve models.mappers.AdminOperatorLogMapper.insert-Inline
### The error occurred while setting parameters
### SQL: insert into admin_operator_log(admin_account,log_type,message,operator_time) values(?,?,?,?)
### Cause: java.sql.SQLException: Data truncated for column 'log_type' at row 1
org.apache.ibatis.exceptions.PersistenceException: 
### Error updating database.  Cause: java.sql.SQLException: Data truncated for column 'log_type' at row 1
### The error may involve models.mappers.AdminOperatorLogMapper.insert-Inline
### The error occurred while setting parameters
### SQL: insert into admin_operator_log(admin_account,log_type,message,operator_time) values(?,?,?,?)
### Cause: java.sql.SQLException: Data truncated for column 'log_type' at row 1
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:26)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:154)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:141)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:51)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:52)
	at com.sun.proxy.$Proxy52.insert(Unknown Source)
	at models.AdminOperatorLog.insert(AdminOperatorLog.java:70)
	at controllers.admins.UserManage.doRecharge(UserManage.java:127)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
Caused by: java.sql.SQLException: Data truncated for column 'log_type' at row 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:998)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3835)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3771)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2435)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2582)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2535)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1911)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1203)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.execute(NewProxyPreparedStatement.java:1392)
	at sun.reflect.GeneratedMethodAccessor25.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.ibatis.logging.jdbc.PreparedStatementLogger.invoke(PreparedStatementLogger.java:62)
	at com.sun.proxy.$Proxy20.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:44)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:69)
	at org.apache.ibatis.executor.ReuseExecutor.doUpdate(ReuseExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:105)
	at org.apache.ibatis.executor.CachingExecutor.update(CachingExecutor.java:71)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:152)
	... 32 more
[2016-08-09 10:45:42,356] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 10:45:42,356] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - 
### Error updating database.  Cause: java.sql.SQLException: Data truncated for column 'log_type' at row 1
### The error may involve models.mappers.AdminOperatorLogMapper.insert-Inline
### The error occurred while setting parameters
### SQL: insert into admin_operator_log(admin_account,log_type,message,operator_time) values(?,?,?,?)
### Cause: java.sql.SQLException: Data truncated for column 'log_type' at row 1
org.apache.ibatis.exceptions.PersistenceException: 
### Error updating database.  Cause: java.sql.SQLException: Data truncated for column 'log_type' at row 1
### The error may involve models.mappers.AdminOperatorLogMapper.insert-Inline
### The error occurred while setting parameters
### SQL: insert into admin_operator_log(admin_account,log_type,message,operator_time) values(?,?,?,?)
### Cause: java.sql.SQLException: Data truncated for column 'log_type' at row 1
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:26)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:154)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:141)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:51)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:52)
	at com.sun.proxy.$Proxy52.insert(Unknown Source)
	at models.AdminOperatorLog.insert(AdminOperatorLog.java:70)
	at controllers.admins.UserManage.doRecharge(UserManage.java:127)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
Caused by: java.sql.SQLException: Data truncated for column 'log_type' at row 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:998)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3835)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3771)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2435)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2582)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2535)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1911)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1203)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.execute(NewProxyPreparedStatement.java:1392)
	at sun.reflect.GeneratedMethodAccessor25.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.ibatis.logging.jdbc.PreparedStatementLogger.invoke(PreparedStatementLogger.java:62)
	at com.sun.proxy.$Proxy20.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:44)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:69)
	at org.apache.ibatis.executor.ReuseExecutor.doUpdate(ReuseExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:105)
	at org.apache.ibatis.executor.CachingExecutor.update(CachingExecutor.java:71)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:152)
	... 32 more
[2016-08-09 10:45:55,532] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 10:45:55,532] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - 
### Error updating database.  Cause: java.sql.SQLException: Data truncated for column 'log_type' at row 1
### The error may involve models.mappers.AdminOperatorLogMapper.insert-Inline
### The error occurred while setting parameters
### SQL: insert into admin_operator_log(admin_account,log_type,message,operator_time) values(?,?,?,?)
### Cause: java.sql.SQLException: Data truncated for column 'log_type' at row 1
org.apache.ibatis.exceptions.PersistenceException: 
### Error updating database.  Cause: java.sql.SQLException: Data truncated for column 'log_type' at row 1
### The error may involve models.mappers.AdminOperatorLogMapper.insert-Inline
### The error occurred while setting parameters
### SQL: insert into admin_operator_log(admin_account,log_type,message,operator_time) values(?,?,?,?)
### Cause: java.sql.SQLException: Data truncated for column 'log_type' at row 1
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:26)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:154)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:141)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:51)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:52)
	at com.sun.proxy.$Proxy52.insert(Unknown Source)
	at models.AdminOperatorLog.insert(AdminOperatorLog.java:70)
	at controllers.admins.UserManage.doRecharge(UserManage.java:127)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
Caused by: java.sql.SQLException: Data truncated for column 'log_type' at row 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:998)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3835)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3771)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2435)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2582)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2535)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1911)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1203)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.execute(NewProxyPreparedStatement.java:1392)
	at sun.reflect.GeneratedMethodAccessor25.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.ibatis.logging.jdbc.PreparedStatementLogger.invoke(PreparedStatementLogger.java:62)
	at com.sun.proxy.$Proxy20.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:44)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:69)
	at org.apache.ibatis.executor.ReuseExecutor.doUpdate(ReuseExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:105)
	at org.apache.ibatis.executor.CachingExecutor.update(CachingExecutor.java:71)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:152)
	... 32 more
[2016-08-09 10:46:46,787] INFO  [jobs-thread-2] jobs.AddKwdForFlow.doJob(AddKwdForFlow.java:38) - =====AddFlow Started=====
[2016-08-09 10:46:46,791] INFO  [jobs-thread-2] jobs.AddKwdForFlow.doJob(AddKwdForFlow.java:40) - 流量任务条数0
[2016-08-09 10:46:49,220] INFO  [jobs-thread-2] jobs.BuyerTaskCancel.doJob(BuyerTaskCancel.java:38) - cancel buyerTask start
[2016-08-09 10:46:49,220] INFO  [jobs-thread-2] jobs.BuyerTaskCancel.doJob(BuyerTaskCancel.java:49) - cancel buyerTask end
[2016-08-09 10:46:49,251] INFO  [jobs-thread-8] jobs.GetExpressNo.doJob(GetExpressNo.java:49) - =====GetExpressNo Started=====
[2016-08-09 10:49:52,443] INFO  [jobs-thread-2] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 10:53:00,832] INFO  [play-thread-1] controllers.UserCenter.checkBeforeSaveBuyerAccount2(UserCenter.java:1048) - Buyer=1704-zhangsan save buyerAccount: Check Duplicate
[2016-08-09 10:54:29,394] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 10:56:24,938] INFO  [main] play.Logger.info(Logger.java:308) - Starting C:\Users\Administrator.SC-201606081350\Desktop\wugake\wgk
[2016-08-09 10:56:25,862] WARN  [main] play.Logger.warn(Logger.java:367) - You're running Play! in DEV mode
[2016-08-09 10:56:25,953] INFO  [main] play.Logger.info(Logger.java:308) - Listening for HTTP on port 80 (Waiting a first request to start) ...
[2016-08-09 10:56:38,608] INFO  [MLog-Init-Reporter] com.mchange.v2.log.MLog$1.run(MLog.java:124) - MLog clients using log4j logging.
[2016-08-09 10:56:38,744] INFO  [play-thread-1] com.mchange.v2.c3p0.C3P0Registry.banner(C3P0Registry.java:248) - Initializing c3p0-0.9.5 [built 02-January-2015 13:25:04 -0500; debug? true; trace: 10]
[2016-08-09 10:56:38,824] INFO  [play-thread-1] com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getPoolManager(AbstractPoolBackedDataSource.java:522) - Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 10, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 1000, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hge10a9iub2mfh1agah0a|141e00a, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceUseNamedDriverClass -> false, identityToken -> 1hge10a9iub2mfh1agah0a|141e00a, idleConnectionTestPeriod -> 10, initialPoolSize -> 3, jdbcUrl -> jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 30, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 10, numHelperThreads -> 3, preferredTestQuery -> /* ping */ SELECT 1, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> true, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
[2016-08-09 10:56:38,844] WARN  [play-thread-1] com.mchange.v2.resourcepool.BasicResourcePool.<init>(BasicResourcePool.java:288) - Bad pool size config, start 3 < min 10. Using 10 as start.
[2016-08-09 10:56:38,884] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Connected to jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8 for default
[2016-08-09 10:56:39,654] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Application 'wgk' is now started !
[2016-08-09 10:56:39,860] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 11:01:36,572] INFO  [play-thread-1] controllers.UserCenter.checkBeforeSaveBuyerAccount2(UserCenter.java:1048) - Buyer=1704-zhangsan save buyerAccount: Check Duplicate
[2016-08-09 11:02:19,342] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 11:02:19,342] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - 
### Error querying database.  Cause: java.lang.IllegalArgumentException: No enum const class enums.Platform.WEIBO
### The error may exist in models/mappers/BuyerAccountMapper.xml
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### SQL: select   ba.id, ba.nick, ba.platform, ba.state, ba.city, ba.region, ba.address,   ba.mobile, ba.status, ba.consignee,ba.memo, ba.create_time, ba.modify_time      ,u.nick userNick     from buyer_account ba      join user u on u.id = ba.user_id      WHERE  ba.status = ?    order by order_number      limit ?,?
### Cause: java.lang.IllegalArgumentException: No enum const class enums.Platform.WEIBO
org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: java.lang.IllegalArgumentException: No enum const class enums.Platform.WEIBO
### The error may exist in models/mappers/BuyerAccountMapper.xml
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### SQL: select   ba.id, ba.nick, ba.platform, ba.state, ba.city, ba.region, ba.address,   ba.mobile, ba.status, ba.consignee,ba.memo, ba.create_time, ba.modify_time      ,u.nick userNick     from buyer_account ba      join user u on u.id = ba.user_id      WHERE  ba.status = ?    order by order_number      limit ?,?
### Cause: java.lang.IllegalArgumentException: No enum const class enums.Platform.WEIBO
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:26)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:111)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:102)
	at org.apache.ibatis.binding.MapperMethod.executeForMany(MapperMethod.java:119)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:63)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:52)
	at com.sun.proxy.$Proxy47.selectBySearch(Unknown Source)
	at models.BuyerAccount.findForPage(BuyerAccount.java:181)
	at controllers.admins.UserManage.fetchAndRenderBuyerAccountPage(UserManage.java:263)
	at controllers.admins.UserManage.listBuyerAccountWaitEaxmine(UserManage.java:251)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
Caused by: java.lang.IllegalArgumentException: No enum const class enums.Platform.WEIBO
	at java.lang.Enum.valueOf(Enum.java:196)
	at org.apache.ibatis.type.EnumTypeHandler.getNullableResult(EnumTypeHandler.java:47)
	at org.apache.ibatis.type.EnumTypeHandler.getNullableResult(EnumTypeHandler.java:26)
	at org.apache.ibatis.type.BaseTypeHandler.getResult(BaseTypeHandler.java:55)
	at org.apache.ibatis.executor.resultset.DefaultResultSetHandler.applyAutomaticMappings(DefaultResultSetHandler.java:413)
	at org.apache.ibatis.executor.resultset.DefaultResultSetHandler.getRowValue(DefaultResultSetHandler.java:336)
	at org.apache.ibatis.executor.resultset.DefaultResultSetHandler.handleRowValuesForSimpleResultMap(DefaultResultSetHandler.java:291)
	at org.apache.ibatis.executor.resultset.DefaultResultSetHandler.handleRowValues(DefaultResultSetHandler.java:266)
	at org.apache.ibatis.executor.resultset.DefaultResultSetHandler.handleResultSet(DefaultResultSetHandler.java:236)
	at org.apache.ibatis.executor.resultset.DefaultResultSetHandler.handleResultSets(DefaultResultSetHandler.java:150)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.query(PreparedStatementHandler.java:60)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.query(RoutingStatementHandler.java:73)
	at org.apache.ibatis.executor.ReuseExecutor.doQuery(ReuseExecutor.java:57)
	at org.apache.ibatis.executor.BaseExecutor.queryFromDatabase(BaseExecutor.java:267)
	at org.apache.ibatis.executor.BaseExecutor.query(BaseExecutor.java:137)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:96)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:77)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:108)
	... 34 more
[2016-08-09 14:54:47,091] INFO  [main] play.Logger.info(Logger.java:308) - Starting D:\myeclipsedoc\wgk
[2016-08-09 14:54:47,964] WARN  [main] play.Logger.warn(Logger.java:367) - You're running Play! in DEV mode
[2016-08-09 14:54:48,040] INFO  [main] play.Logger.info(Logger.java:308) - Listening for HTTP on port 80 (Waiting a first request to start) ...
[2016-08-09 14:54:58,328] INFO  [MLog-Init-Reporter] com.mchange.v2.log.MLog$1.run(MLog.java:124) - MLog clients using log4j logging.
[2016-08-09 14:54:58,419] INFO  [play-thread-1] com.mchange.v2.c3p0.C3P0Registry.banner(C3P0Registry.java:248) - Initializing c3p0-0.9.5 [built 02-January-2015 13:25:04 -0500; debug? true; trace: 10]
[2016-08-09 14:54:58,489] INFO  [play-thread-1] com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getPoolManager(AbstractPoolBackedDataSource.java:522) - Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 10, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 1000, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hge10a9iujl45dsy5jcz|550a6723, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceUseNamedDriverClass -> false, identityToken -> 1hge10a9iujl45dsy5jcz|550a6723, idleConnectionTestPeriod -> 10, initialPoolSize -> 3, jdbcUrl -> jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 30, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 10, numHelperThreads -> 3, preferredTestQuery -> /* ping */ SELECT 1, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> true, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
[2016-08-09 14:54:58,509] WARN  [play-thread-1] com.mchange.v2.resourcepool.BasicResourcePool.<init>(BasicResourcePool.java:288) - Bad pool size config, start 3 < min 10. Using 10 as start.
[2016-08-09 14:54:58,549] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Connected to jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8 for default
[2016-08-09 14:54:59,131] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Application 'wgk' is now started !
[2016-08-09 14:54:59,321] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 14:55:15,977] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@712o35f5i
Internal Server Error (500) for request POST /doLogin

Execution exception (In /app/models/Task.java around line 876)
NullPointerException occured : Try to read platform on null object models.Task (models.Task.isTaoBaoAndTmall, line 876)

play.exceptions.JavaExecutionException: Try to read platform on null object models.Task (models.Task.isTaoBaoAndTmall, line 876)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:228)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.NullPointerException: Try to read platform on null object models.Task (models.Task.isTaoBaoAndTmall, line 876)
	at play.classloading.enhancers.PropertiesEnhancer$FieldAccessor.invokeReadProperty(PropertiesEnhancer.java:214)
	at models.Task.isTaoBaoAndTmall(Task.java:876)
	at models.BuyerTask.findTodoTask(BuyerTask.java:1234)
	at controllers.UserAuthentication.setUserTodoTask(UserAuthentication.java:381)
	at controllers.UserAuthentication.doLogin(UserAuthentication.java:368)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	... 1 more
[2016-08-09 15:13:09,425] INFO  [main] play.Logger.info(Logger.java:308) - Starting D:\myeclipsedoc\wgk
[2016-08-09 15:13:10,521] WARN  [main] play.Logger.warn(Logger.java:367) - You're running Play! in DEV mode
[2016-08-09 15:13:10,639] INFO  [main] play.Logger.info(Logger.java:308) - Listening for HTTP on port 80 (Waiting a first request to start) ...
[2016-08-09 15:13:24,292] INFO  [MLog-Init-Reporter] com.mchange.v2.log.MLog$1.run(MLog.java:124) - MLog clients using log4j logging.
[2016-08-09 15:13:24,433] INFO  [play-thread-1] com.mchange.v2.c3p0.C3P0Registry.banner(C3P0Registry.java:248) - Initializing c3p0-0.9.5 [built 02-January-2015 13:25:04 -0500; debug? true; trace: 10]
[2016-08-09 15:13:24,527] INFO  [play-thread-1] com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getPoolManager(AbstractPoolBackedDataSource.java:522) - Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 10, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 1000, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hge10a9iuk8tictp31go|13a66c87, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceUseNamedDriverClass -> false, identityToken -> 1hge10a9iuk8tictp31go|13a66c87, idleConnectionTestPeriod -> 10, initialPoolSize -> 3, jdbcUrl -> jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 30, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 10, numHelperThreads -> 3, preferredTestQuery -> /* ping */ SELECT 1, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> true, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
[2016-08-09 15:13:24,542] WARN  [play-thread-1] com.mchange.v2.resourcepool.BasicResourcePool.<init>(BasicResourcePool.java:288) - Bad pool size config, start 3 < min 10. Using 10 as start.
[2016-08-09 15:13:24,605] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Connected to jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8 for default
[2016-08-09 15:13:25,291] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Application 'wgk' is now started !
[2016-08-09 15:13:25,525] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 15:13:28,036] ERROR [play-thread-1] com.aton.base.secure.Secure.checkAuthentication(Secure.java:92) - Login admin not exists,admin_id=null
[2016-08-09 15:15:45,212] INFO  [play-thread-1] controllers.UserCenter.checkBeforeSaveBuyerAccount2(UserCenter.java:1048) - Buyer=1678-maimai save buyerAccount: Check Duplicate
[2016-08-09 15:15:45,228] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 15:15:45,228] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - 
### Error updating database.  Cause: java.sql.SQLException: Data truncated for column 'platform' at row 1
### The error may involve models.mappers.BuyerAccountMapper2.insert-Inline
### The error occurred while setting parameters
### SQL: insert into buyer_account(nick,platform,consignee,state,city,region,address,mobile,user_id,status,create_time,modify_time) values(?,?,?,?,?,?,?,?,?,?,?,?)
### Cause: java.sql.SQLException: Data truncated for column 'platform' at row 1
org.apache.ibatis.exceptions.PersistenceException: 
### Error updating database.  Cause: java.sql.SQLException: Data truncated for column 'platform' at row 1
### The error may involve models.mappers.BuyerAccountMapper2.insert-Inline
### The error occurred while setting parameters
### SQL: insert into buyer_account(nick,platform,consignee,state,city,region,address,mobile,user_id,status,create_time,modify_time) values(?,?,?,?,?,?,?,?,?,?,?,?)
### Cause: java.sql.SQLException: Data truncated for column 'platform' at row 1
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:26)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:154)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:141)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:51)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:52)
	at com.sun.proxy.$Proxy47.insert(Unknown Source)
	at models.BuyerAccount2.save(BuyerAccount2.java:236)
	at controllers.UserCenter.saveBuyerAccount2(UserCenter.java:1026)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
Caused by: java.sql.SQLException: Data truncated for column 'platform' at row 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:998)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3835)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3771)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2435)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2582)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2535)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1911)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1203)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.execute(NewProxyPreparedStatement.java:1392)
	at sun.reflect.GeneratedMethodAccessor31.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.ibatis.logging.jdbc.PreparedStatementLogger.invoke(PreparedStatementLogger.java:62)
	at com.sun.proxy.$Proxy20.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:44)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:69)
	at org.apache.ibatis.executor.ReuseExecutor.doUpdate(ReuseExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:105)
	at org.apache.ibatis.executor.CachingExecutor.update(CachingExecutor.java:71)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:152)
	... 32 more
[2016-08-09 15:17:29,790] INFO  [play-thread-1] controllers.UserCenter.checkBeforeSaveBuyerAccount2(UserCenter.java:1048) - Buyer=1678-maimai save buyerAccount: Check Duplicate
[2016-08-09 15:18:01,153] INFO  [play-thread-1] controllers.UserCenter.checkBeforeSaveBuyerAccount2(UserCenter.java:1048) - Buyer=1678-maimai save buyerAccount: Check Duplicate
[2016-08-09 15:18:01,153] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 15:18:01,153] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - 
### Error updating database.  Cause: java.sql.SQLException: Data truncated for column 'platform' at row 1
### The error may involve models.mappers.BuyerAccountMapper2.insert-Inline
### The error occurred while setting parameters
### SQL: insert into buyer_account(nick,platform,consignee,state,city,region,address,mobile,user_id,status,create_time,modify_time) values(?,?,?,?,?,?,?,?,?,?,?,?)
### Cause: java.sql.SQLException: Data truncated for column 'platform' at row 1
org.apache.ibatis.exceptions.PersistenceException: 
### Error updating database.  Cause: java.sql.SQLException: Data truncated for column 'platform' at row 1
### The error may involve models.mappers.BuyerAccountMapper2.insert-Inline
### The error occurred while setting parameters
### SQL: insert into buyer_account(nick,platform,consignee,state,city,region,address,mobile,user_id,status,create_time,modify_time) values(?,?,?,?,?,?,?,?,?,?,?,?)
### Cause: java.sql.SQLException: Data truncated for column 'platform' at row 1
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:26)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:154)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:141)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:51)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:52)
	at com.sun.proxy.$Proxy47.insert(Unknown Source)
	at models.BuyerAccount2.save(BuyerAccount2.java:236)
	at controllers.UserCenter.saveBuyerAccount2(UserCenter.java:1026)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
Caused by: java.sql.SQLException: Data truncated for column 'platform' at row 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:998)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3835)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3771)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2435)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2582)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2535)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1911)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1203)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.execute(NewProxyPreparedStatement.java:1392)
	at sun.reflect.GeneratedMethodAccessor31.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.ibatis.logging.jdbc.PreparedStatementLogger.invoke(PreparedStatementLogger.java:62)
	at com.sun.proxy.$Proxy20.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:44)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:69)
	at org.apache.ibatis.executor.ReuseExecutor.doUpdate(ReuseExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:105)
	at org.apache.ibatis.executor.CachingExecutor.update(CachingExecutor.java:71)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:152)
	... 32 more
[2016-08-09 15:18:21,251] INFO  [play-thread-1] controllers.UserCenter.checkBeforeSaveBuyerAccount2(UserCenter.java:1048) - Buyer=1678-maimai save buyerAccount: Check Duplicate
[2016-08-09 15:22:27,764] INFO  [play-thread-1] controllers.UserCenter.checkBeforeSaveBuyerAccount2(UserCenter.java:1048) - Buyer=1678-maimai save buyerAccount: Check Duplicate
[2016-08-09 15:22:27,774] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 15:22:27,774] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - 
### Error updating database.  Cause: java.sql.SQLException: Data truncated for column 'platform' at row 1
### The error may involve models.mappers.BuyerAccountMapper2.insert-Inline
### The error occurred while setting parameters
### SQL: insert into buyer_account(nick,platform,consignee,state,city,region,address,mobile,user_id,status,create_time,modify_time) values(?,?,?,?,?,?,?,?,?,?,?,?)
### Cause: java.sql.SQLException: Data truncated for column 'platform' at row 1
org.apache.ibatis.exceptions.PersistenceException: 
### Error updating database.  Cause: java.sql.SQLException: Data truncated for column 'platform' at row 1
### The error may involve models.mappers.BuyerAccountMapper2.insert-Inline
### The error occurred while setting parameters
### SQL: insert into buyer_account(nick,platform,consignee,state,city,region,address,mobile,user_id,status,create_time,modify_time) values(?,?,?,?,?,?,?,?,?,?,?,?)
### Cause: java.sql.SQLException: Data truncated for column 'platform' at row 1
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:26)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:154)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:141)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:51)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:52)
	at com.sun.proxy.$Proxy47.insert(Unknown Source)
	at models.BuyerAccount2.save(BuyerAccount2.java:236)
	at controllers.UserCenter.saveBuyerAccount2(UserCenter.java:1026)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
Caused by: java.sql.SQLException: Data truncated for column 'platform' at row 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:998)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3835)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3771)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2435)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2582)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2535)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1911)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1203)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.execute(NewProxyPreparedStatement.java:1392)
	at sun.reflect.GeneratedMethodAccessor31.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.ibatis.logging.jdbc.PreparedStatementLogger.invoke(PreparedStatementLogger.java:62)
	at com.sun.proxy.$Proxy20.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:44)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:69)
	at org.apache.ibatis.executor.ReuseExecutor.doUpdate(ReuseExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:105)
	at org.apache.ibatis.executor.CachingExecutor.update(CachingExecutor.java:71)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:152)
	... 32 more
[2016-08-09 15:22:33,995] INFO  [play-thread-1] controllers.UserCenter.checkBeforeSaveBuyerAccount2(UserCenter.java:1048) - Buyer=1678-maimai save buyerAccount: Check Duplicate
[2016-08-09 15:22:33,995] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 15:22:33,995] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - 
### Error updating database.  Cause: java.sql.SQLException: Data truncated for column 'platform' at row 1
### The error may involve models.mappers.BuyerAccountMapper2.insert-Inline
### The error occurred while setting parameters
### SQL: insert into buyer_account(nick,platform,consignee,state,city,region,address,mobile,user_id,status,create_time,modify_time) values(?,?,?,?,?,?,?,?,?,?,?,?)
### Cause: java.sql.SQLException: Data truncated for column 'platform' at row 1
org.apache.ibatis.exceptions.PersistenceException: 
### Error updating database.  Cause: java.sql.SQLException: Data truncated for column 'platform' at row 1
### The error may involve models.mappers.BuyerAccountMapper2.insert-Inline
### The error occurred while setting parameters
### SQL: insert into buyer_account(nick,platform,consignee,state,city,region,address,mobile,user_id,status,create_time,modify_time) values(?,?,?,?,?,?,?,?,?,?,?,?)
### Cause: java.sql.SQLException: Data truncated for column 'platform' at row 1
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:26)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:154)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:141)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:51)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:52)
	at com.sun.proxy.$Proxy47.insert(Unknown Source)
	at models.BuyerAccount2.save(BuyerAccount2.java:236)
	at controllers.UserCenter.saveBuyerAccount2(UserCenter.java:1026)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
Caused by: java.sql.SQLException: Data truncated for column 'platform' at row 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:998)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3835)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3771)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2435)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2582)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2535)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1911)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1203)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.execute(NewProxyPreparedStatement.java:1392)
	at sun.reflect.GeneratedMethodAccessor31.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.ibatis.logging.jdbc.PreparedStatementLogger.invoke(PreparedStatementLogger.java:62)
	at com.sun.proxy.$Proxy20.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:44)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:69)
	at org.apache.ibatis.executor.ReuseExecutor.doUpdate(ReuseExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:105)
	at org.apache.ibatis.executor.CachingExecutor.update(CachingExecutor.java:71)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:152)
	... 32 more
[2016-08-09 15:23:25,409] INFO  [main] play.Logger.info(Logger.java:308) - Starting D:\myeclipsedoc\wgk
[2016-08-09 15:23:26,173] WARN  [main] play.Logger.warn(Logger.java:367) - You're running Play! in DEV mode
[2016-08-09 15:23:26,236] INFO  [main] play.Logger.info(Logger.java:308) - Listening for HTTP on port 80 (Waiting a first request to start) ...
[2016-08-09 15:24:00,197] INFO  [MLog-Init-Reporter] com.mchange.v2.log.MLog$1.run(MLog.java:124) - MLog clients using log4j logging.
[2016-08-09 15:24:00,291] INFO  [play-thread-1] com.mchange.v2.c3p0.C3P0Registry.banner(C3P0Registry.java:248) - Initializing c3p0-0.9.5 [built 02-January-2015 13:25:04 -0500; debug? true; trace: 10]
[2016-08-09 15:24:00,369] INFO  [play-thread-1] com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getPoolManager(AbstractPoolBackedDataSource.java:522) - Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 10, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 1000, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hge10a9iukmg6tc2cmvc|5a74b10b, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceUseNamedDriverClass -> false, identityToken -> 1hge10a9iukmg6tc2cmvc|5a74b10b, idleConnectionTestPeriod -> 10, initialPoolSize -> 3, jdbcUrl -> jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 30, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 10, numHelperThreads -> 3, preferredTestQuery -> /* ping */ SELECT 1, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> true, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
[2016-08-09 15:24:00,385] WARN  [play-thread-1] com.mchange.v2.resourcepool.BasicResourcePool.<init>(BasicResourcePool.java:288) - Bad pool size config, start 3 < min 10. Using 10 as start.
[2016-08-09 15:24:00,416] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Connected to jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8 for default
[2016-08-09 15:24:01,009] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Application 'wgk' is now started !
[2016-08-09 15:24:01,259] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 15:24:29,126] INFO  [play-thread-1] controllers.UserCenter.checkBeforeSaveBuyerAccount2(UserCenter.java:1048) - Buyer=1678-maimai save buyerAccount: Check Duplicate
[2016-08-09 15:24:40,665] INFO  [play-thread-1] controllers.UserCenter.checkBeforeSaveBuyerAccount2(UserCenter.java:1048) - Buyer=1678-maimai save buyerAccount: Check Duplicate
[2016-08-09 15:26:45,340] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 15:26:55,991] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 15:26:55,991] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - Try to read taskRequest on null object models.Task (controllers.TaskExecutor.getTaskRequest, line 947)
java.lang.NullPointerException: Try to read taskRequest on null object models.Task (controllers.TaskExecutor.getTaskRequest, line 947)
	at play.classloading.enhancers.PropertiesEnhancer$FieldAccessor.invokeReadProperty(PropertiesEnhancer.java:214)
	at controllers.TaskExecutor.getTaskRequest(TaskExecutor.java:947)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-08-09 15:29:47,207] ERROR [jobs-thread-2] play.Logger.niceThrowable(Logger.java:604) - 

@712o7jb84
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/controllers/TaskExecutor.java around line 945)
The file /app/controllers/TaskExecutor.java could not be compiled. Error raised is : Duplicate method getTaskRequest(long) in type TaskExecutor

play.exceptions.CompilationException: Duplicate method getTaskRequest(long) in type TaskExecutor
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:480)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-08-09 15:30:25,752] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 15:32:27,139] ERROR [jobs-thread-4] play.Logger.niceThrowable(Logger.java:604) - 

@712o7jb86
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/controllers/TaskExecutor.java around line 652)
The file /app/controllers/TaskExecutor.java could not be compiled. Error raised is : Duplicate method take(BuyerTask) in type TaskExecutor

play.exceptions.CompilationException: Duplicate method take(BuyerTask) in type TaskExecutor
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:480)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-08-09 15:34:25,947] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 15:36:10,132] ERROR [play-thread-1] com.aton.base.secure.Secure.checkAuthentication(Secure.java:92) - Login admin not exists,admin_id=null
[2016-08-09 15:43:17,866] INFO  [main] play.Logger.info(Logger.java:308) - Starting D:\myeclipsedoc\wgk
[2016-08-09 15:43:18,740] WARN  [main] play.Logger.warn(Logger.java:367) - You're running Play! in DEV mode
[2016-08-09 15:43:18,834] INFO  [main] play.Logger.info(Logger.java:308) - Listening for HTTP on port 80 (Waiting a first request to start) ...
[2016-08-09 15:44:02,483] INFO  [MLog-Init-Reporter] com.mchange.v2.log.MLog$1.run(MLog.java:124) - MLog clients using log4j logging.
[2016-08-09 15:44:02,612] INFO  [play-thread-1] com.mchange.v2.c3p0.C3P0Registry.banner(C3P0Registry.java:248) - Initializing c3p0-0.9.5 [built 02-January-2015 13:25:04 -0500; debug? true; trace: 10]
[2016-08-09 15:44:02,675] INFO  [play-thread-1] com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getPoolManager(AbstractPoolBackedDataSource.java:522) - Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 10, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 1000, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hge10a9iulc7vahtevpu|62d5163a, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceUseNamedDriverClass -> false, identityToken -> 1hge10a9iulc7vahtevpu|62d5163a, idleConnectionTestPeriod -> 10, initialPoolSize -> 3, jdbcUrl -> jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 30, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 10, numHelperThreads -> 3, preferredTestQuery -> /* ping */ SELECT 1, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> true, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
[2016-08-09 15:44:02,691] WARN  [play-thread-1] com.mchange.v2.resourcepool.BasicResourcePool.<init>(BasicResourcePool.java:288) - Bad pool size config, start 3 < min 10. Using 10 as start.
[2016-08-09 15:44:02,741] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Connected to jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8 for default
[2016-08-09 15:44:03,556] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Application 'wgk' is now started !
[2016-08-09 15:44:03,743] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 15:44:06,689] ERROR [play-thread-1] com.aton.base.secure.Secure.checkAuthentication(Secure.java:92) - Login admin not exists,admin_id=null
[2016-08-09 15:46:06,941] ERROR [jobs-thread-7] play.Logger.niceThrowable(Logger.java:604) - 

@712o9n4g6
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/controllers/Admin.java around line 754)
The file /app/controllers/Admin.java could not be compiled. Error raised is : Duplicate method buyerAccount() in type Admin

play.exceptions.CompilationException: Duplicate method buyerAccount() in type Admin
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:480)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-08-09 15:49:04,696] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 15:56:07,376] ERROR [jobs-thread-4] play.Logger.niceThrowable(Logger.java:604) - 

@712o9n4g8
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/controllers/admins/UserManage.java around line 234)
The file /app/controllers/admins/UserManage.java could not be compiled. Error raised is : Duplicate method listBuyerAccount(BuyerAccountSearchVo) in type UserManage

play.exceptions.CompilationException: Duplicate method listBuyerAccount(BuyerAccountSearchVo) in type UserManage
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:480)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-08-09 15:58:15,976] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 16:00:00,798] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 16:00:00,798] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - 
### Error querying database.  Cause: java.lang.IllegalArgumentException: No enum const class enums.Platform.QQ
### The error may exist in models/mappers/BuyerAccountMapper.xml
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### SQL: select   ba.id, ba.nick, ba.platform, ba.state, ba.city, ba.region, ba.address,   ba.mobile, ba.status, ba.consignee,ba.memo, ba.create_time, ba.modify_time      ,u.nick userNick     from buyer_account ba      join user u on u.id = ba.user_id      WHERE  ba.status = ?             and u.nick like '%%'          and ba.nick like '%%'    order by order_number      limit ?,?
### Cause: java.lang.IllegalArgumentException: No enum const class enums.Platform.QQ
org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: java.lang.IllegalArgumentException: No enum const class enums.Platform.QQ
### The error may exist in models/mappers/BuyerAccountMapper.xml
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### SQL: select   ba.id, ba.nick, ba.platform, ba.state, ba.city, ba.region, ba.address,   ba.mobile, ba.status, ba.consignee,ba.memo, ba.create_time, ba.modify_time      ,u.nick userNick     from buyer_account ba      join user u on u.id = ba.user_id      WHERE  ba.status = ?             and u.nick like '%%'          and ba.nick like '%%'    order by order_number      limit ?,?
### Cause: java.lang.IllegalArgumentException: No enum const class enums.Platform.QQ
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:26)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:111)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:102)
	at org.apache.ibatis.binding.MapperMethod.executeForMany(MapperMethod.java:119)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:63)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:52)
	at com.sun.proxy.$Proxy61.selectBySearch(Unknown Source)
	at models.BuyerAccount.findForPage(BuyerAccount.java:181)
	at controllers.admins.UserManage.fetchAndRenderBuyerAccountPage(UserManage.java:286)
	at controllers.admins.UserManage.listBuyerAccount(UserManage.java:237)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
Caused by: java.lang.IllegalArgumentException: No enum const class enums.Platform.QQ
	at java.lang.Enum.valueOf(Enum.java:196)
	at org.apache.ibatis.type.EnumTypeHandler.getNullableResult(EnumTypeHandler.java:47)
	at org.apache.ibatis.type.EnumTypeHandler.getNullableResult(EnumTypeHandler.java:26)
	at org.apache.ibatis.type.BaseTypeHandler.getResult(BaseTypeHandler.java:55)
	at org.apache.ibatis.executor.resultset.DefaultResultSetHandler.applyAutomaticMappings(DefaultResultSetHandler.java:413)
	at org.apache.ibatis.executor.resultset.DefaultResultSetHandler.getRowValue(DefaultResultSetHandler.java:336)
	at org.apache.ibatis.executor.resultset.DefaultResultSetHandler.handleRowValuesForSimpleResultMap(DefaultResultSetHandler.java:291)
	at org.apache.ibatis.executor.resultset.DefaultResultSetHandler.handleRowValues(DefaultResultSetHandler.java:266)
	at org.apache.ibatis.executor.resultset.DefaultResultSetHandler.handleResultSet(DefaultResultSetHandler.java:236)
	at org.apache.ibatis.executor.resultset.DefaultResultSetHandler.handleResultSets(DefaultResultSetHandler.java:150)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.query(PreparedStatementHandler.java:60)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.query(RoutingStatementHandler.java:73)
	at org.apache.ibatis.executor.ReuseExecutor.doQuery(ReuseExecutor.java:57)
	at org.apache.ibatis.executor.BaseExecutor.queryFromDatabase(BaseExecutor.java:267)
	at org.apache.ibatis.executor.BaseExecutor.query(BaseExecutor.java:137)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:96)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:77)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:108)
	... 34 more
[2016-08-09 16:00:21,574] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-08-09 16:00:21,574] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - 
### Error querying database.  Cause: java.lang.IllegalArgumentException: No enum const class enums.Platform.QQ
### The error may exist in models/mappers/BuyerAccountMapper.xml
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### SQL: select   ba.id, ba.nick, ba.platform, ba.state, ba.city, ba.region, ba.address,   ba.mobile, ba.status, ba.consignee,ba.memo, ba.create_time, ba.modify_time      ,u.nick userNick     from buyer_account ba      join user u on u.id = ba.user_id      WHERE  ba.status = ?             and u.nick like '%%'          and ba.nick like '%%'    order by order_number      limit ?,?
### Cause: java.lang.IllegalArgumentException: No enum const class enums.Platform.QQ
org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: java.lang.IllegalArgumentException: No enum const class enums.Platform.QQ
### The error may exist in models/mappers/BuyerAccountMapper.xml
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### SQL: select   ba.id, ba.nick, ba.platform, ba.state, ba.city, ba.region, ba.address,   ba.mobile, ba.status, ba.consignee,ba.memo, ba.create_time, ba.modify_time      ,u.nick userNick     from buyer_account ba      join user u on u.id = ba.user_id      WHERE  ba.status = ?             and u.nick like '%%'          and ba.nick like '%%'    order by order_number      limit ?,?
### Cause: java.lang.IllegalArgumentException: No enum const class enums.Platform.QQ
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:26)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:111)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:102)
	at org.apache.ibatis.binding.MapperMethod.executeForMany(MapperMethod.java:119)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:63)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:52)
	at com.sun.proxy.$Proxy61.selectBySearch(Unknown Source)
	at models.BuyerAccount.findForPage(BuyerAccount.java:181)
	at controllers.admins.UserManage.fetchAndRenderBuyerAccountPage(UserManage.java:286)
	at controllers.admins.UserManage.listBuyerAccount(UserManage.java:237)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
Caused by: java.lang.IllegalArgumentException: No enum const class enums.Platform.QQ
	at java.lang.Enum.valueOf(Enum.java:196)
	at org.apache.ibatis.type.EnumTypeHandler.getNullableResult(EnumTypeHandler.java:47)
	at org.apache.ibatis.type.EnumTypeHandler.getNullableResult(EnumTypeHandler.java:26)
	at org.apache.ibatis.type.BaseTypeHandler.getResult(BaseTypeHandler.java:55)
	at org.apache.ibatis.executor.resultset.DefaultResultSetHandler.applyAutomaticMappings(DefaultResultSetHandler.java:413)
	at org.apache.ibatis.executor.resultset.DefaultResultSetHandler.getRowValue(DefaultResultSetHandler.java:336)
	at org.apache.ibatis.executor.resultset.DefaultResultSetHandler.handleRowValuesForSimpleResultMap(DefaultResultSetHandler.java:291)
	at org.apache.ibatis.executor.resultset.DefaultResultSetHandler.handleRowValues(DefaultResultSetHandler.java:266)
	at org.apache.ibatis.executor.resultset.DefaultResultSetHandler.handleResultSet(DefaultResultSetHandler.java:236)
	at org.apache.ibatis.executor.resultset.DefaultResultSetHandler.handleResultSets(DefaultResultSetHandler.java:150)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.query(PreparedStatementHandler.java:60)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.query(RoutingStatementHandler.java:73)
	at org.apache.ibatis.executor.ReuseExecutor.doQuery(ReuseExecutor.java:57)
	at org.apache.ibatis.executor.BaseExecutor.queryFromDatabase(BaseExecutor.java:267)
	at org.apache.ibatis.executor.BaseExecutor.query(BaseExecutor.java:137)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:96)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:77)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:108)
	... 34 more
[2016-08-09 16:08:15,850] INFO  [jobs-thread-6] jobs.AddKwdForFlow.doJob(AddKwdForFlow.java:38) - =====AddFlow Started=====
[2016-08-09 16:08:15,866] INFO  [jobs-thread-6] jobs.AddKwdForFlow.doJob(AddKwdForFlow.java:40) - 流量任务条数0
[2016-08-09 16:08:17,021] INFO  [jobs-thread-6] jobs.BuyerTaskCancel.doJob(BuyerTaskCancel.java:38) - cancel buyerTask start
[2016-08-09 16:08:17,037] INFO  [jobs-thread-6] jobs.BuyerTaskCancel.doJob(BuyerTaskCancel.java:49) - cancel buyerTask end
[2016-08-09 16:08:17,053] INFO  [jobs-thread-7] jobs.GetExpressNo.doJob(GetExpressNo.java:49) - =====GetExpressNo Started=====
[2016-08-09 16:08:19,128] INFO  [jobs-thread-7] com.aton.util.FaBaoGuoUtil.execute(FaBaoGuoUtil.java:63) - execute is faild!!
[2016-08-09 16:08:19,144] INFO  [jobs-thread-7] jobs.GetExpressNo.doJob(GetExpressNo.java:59) - fa bao guo return null!
[2016-08-09 16:11:18,064] INFO  [jobs-thread-6] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 16:12:19,496] INFO  [jobs-thread-2] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 16:19:17,747] INFO  [main] play.Logger.info(Logger.java:308) - Starting D:\myeclipsedoc\wgk
[2016-08-09 16:19:18,481] WARN  [main] play.Logger.warn(Logger.java:367) - You're running Play! in DEV mode
[2016-08-09 16:19:18,560] INFO  [main] play.Logger.info(Logger.java:308) - Listening for HTTP on port 80 (Waiting a first request to start) ...
[2016-08-09 16:26:53,272] INFO  [MLog-Init-Reporter] com.mchange.v2.log.MLog$1.run(MLog.java:124) - MLog clients using log4j logging.
[2016-08-09 16:26:53,348] INFO  [play-thread-1] com.mchange.v2.c3p0.C3P0Registry.banner(C3P0Registry.java:248) - Initializing c3p0-0.9.5 [built 02-January-2015 13:25:04 -0500; debug? true; trace: 10]
[2016-08-09 16:26:53,415] INFO  [play-thread-1] com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getPoolManager(AbstractPoolBackedDataSource.java:522) - Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 10, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 1000, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hge10a9iumvbi41ioxdu4|23b561a2, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceUseNamedDriverClass -> false, identityToken -> 1hge10a9iumvbi41ioxdu4|23b561a2, idleConnectionTestPeriod -> 10, initialPoolSize -> 3, jdbcUrl -> jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 30, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 10, numHelperThreads -> 3, preferredTestQuery -> /* ping */ SELECT 1, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> true, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
[2016-08-09 16:26:53,431] WARN  [play-thread-1] com.mchange.v2.resourcepool.BasicResourcePool.<init>(BasicResourcePool.java:288) - Bad pool size config, start 3 < min 10. Using 10 as start.
[2016-08-09 16:26:53,466] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Connected to jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8 for default
[2016-08-09 16:26:54,193] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Application 'wgk' is now started !
[2016-08-09 16:26:54,405] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 16:33:31,614] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 16:36:33,929] ERROR [jobs-thread-4] play.Logger.niceThrowable(Logger.java:604) - 

@712ogdaak
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/controllers/Admin.java around line 559)
The file /app/controllers/Admin.java could not be compiled. Error raised is : Duplicate method examineBuyerAccount(BuyerAccount) in type Admin

play.exceptions.CompilationException: Duplicate method examineBuyerAccount(BuyerAccount) in type Admin
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:480)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-08-09 16:38:03,048] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 16:42:34,413] INFO  [play-thread-1] controllers.UserCenter.checkBeforeSaveBuyerAccount2(UserCenter.java:1048) - Buyer=1678-maimai save buyerAccount: Check Duplicate
[2016-08-09 16:47:23,913] INFO  [main] play.Logger.info(Logger.java:308) - Starting D:\myeclipsedoc\wgk
[2016-08-09 16:47:24,752] WARN  [main] play.Logger.warn(Logger.java:367) - You're running Play! in DEV mode
[2016-08-09 16:47:24,838] INFO  [main] play.Logger.info(Logger.java:308) - Listening for HTTP on port 80 (Waiting a first request to start) ...
[2016-08-09 16:47:39,682] INFO  [MLog-Init-Reporter] com.mchange.v2.log.MLog$1.run(MLog.java:124) - MLog clients using log4j logging.
[2016-08-09 16:47:39,846] INFO  [play-thread-1] com.mchange.v2.c3p0.C3P0Registry.banner(C3P0Registry.java:248) - Initializing c3p0-0.9.5 [built 02-January-2015 13:25:04 -0500; debug? true; trace: 10]
[2016-08-09 16:47:39,966] INFO  [play-thread-1] com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getPoolManager(AbstractPoolBackedDataSource.java:522) - Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 10, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 1000, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hge10a9iunm19a8pkge|6ec12f3, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceUseNamedDriverClass -> false, identityToken -> 1hge10a9iunm19a8pkge|6ec12f3, idleConnectionTestPeriod -> 10, initialPoolSize -> 3, jdbcUrl -> jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 30, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 10, numHelperThreads -> 3, preferredTestQuery -> /* ping */ SELECT 1, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> true, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
[2016-08-09 16:47:40,007] WARN  [play-thread-1] com.mchange.v2.resourcepool.BasicResourcePool.<init>(BasicResourcePool.java:288) - Bad pool size config, start 3 < min 10. Using 10 as start.
[2016-08-09 16:47:40,064] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Connected to jdbc:mysql://localhost:3306/waguke?autoReconnect=true&useUnicode=true&characterEncoding=utf-8 for default
[2016-08-09 16:47:40,808] INFO  [play-thread-1] play.Logger.info(Logger.java:308) - Application 'wgk' is now started !
[2016-08-09 16:47:41,053] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 16:48:35,879] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@712oi2c62
Internal Server Error (500) for request POST /UserAuthentication/ma?platform=WM&status=PUBLISHED&device=MOBILE

Execution exception (In /app/enums/Platform.java around line 1)
IllegalArgumentException occured : No enum const class enums.Platform.WM

play.exceptions.JavaExecutionException: No enum const class enums.Platform.WM
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:228)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.IllegalArgumentException: No enum const class enums.Platform.WM
	at java.lang.Enum.valueOf(Enum.java:196)
	at enums.Platform.valueOf(Platform.java:1)
	at controllers.UserAuthentication.ma(UserAuthentication.java:125)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	... 1 more
[2016-08-09 16:48:40,017] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@712oi2c63
Internal Server Error (500) for request POST /UserAuthentication/ma?platform=OTHER&status=PUBLISHED&device=MOBILE

Execution exception (In /app/enums/Platform.java around line 1)
IllegalArgumentException occured : No enum const class enums.Platform.OTHER

play.exceptions.JavaExecutionException: No enum const class enums.Platform.OTHER
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:228)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.IllegalArgumentException: No enum const class enums.Platform.OTHER
	at java.lang.Enum.valueOf(Enum.java:196)
	at enums.Platform.valueOf(Platform.java:1)
	at controllers.UserAuthentication.ma(UserAuthentication.java:125)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	... 1 more
[2016-08-09 16:48:55,541] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@712oi2c64
UserAuthentication.listBuyerAccountsApp action not found

Action not found
Action UserAuthentication.listBuyerAccountsApp could not be found. Error raised is No method public static void listBuyerAccountsApp() was found in class controllers.UserAuthentication

play.exceptions.ActionNotFoundException: Action UserAuthentication.listBuyerAccountsApp not found
	at play.mvc.ActionInvoker.getActionMethod(ActionInvoker.java:577)
	at play.mvc.ActionInvoker.resolve(ActionInvoker.java:84)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.Exception: No method public static void listBuyerAccountsApp() was found in class controllers.UserAuthentication
	... 3 more
[2016-08-09 16:49:05,536] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@712oi2c65
UserAuthentication.listBuyerAccountsApp action not found

Action not found
Action UserAuthentication.listBuyerAccountsApp could not be found. Error raised is No method public static void listBuyerAccountsApp() was found in class controllers.UserAuthentication

play.exceptions.ActionNotFoundException: Action UserAuthentication.listBuyerAccountsApp not found
	at play.mvc.ActionInvoker.getActionMethod(ActionInvoker.java:577)
	at play.mvc.ActionInvoker.resolve(ActionInvoker.java:84)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.Exception: No method public static void listBuyerAccountsApp() was found in class controllers.UserAuthentication
	... 3 more
[2016-08-09 16:49:23,713] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@712oi2c66
UserAuthentication.listBuyerAccountsApp action not found

Action not found
Action UserAuthentication.listBuyerAccountsApp could not be found. Error raised is No method public static void listBuyerAccountsApp() was found in class controllers.UserAuthentication

play.exceptions.ActionNotFoundException: Action UserAuthentication.listBuyerAccountsApp not found
	at play.mvc.ActionInvoker.getActionMethod(ActionInvoker.java:577)
	at play.mvc.ActionInvoker.resolve(ActionInvoker.java:84)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.Exception: No method public static void listBuyerAccountsApp() was found in class controllers.UserAuthentication
	... 3 more
[2016-08-09 16:49:31,933] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@712oi2c67
UserAuthentication.listBuyerAccountsApp action not found

Action not found
Action UserAuthentication.listBuyerAccountsApp could not be found. Error raised is No method public static void listBuyerAccountsApp() was found in class controllers.UserAuthentication

play.exceptions.ActionNotFoundException: Action UserAuthentication.listBuyerAccountsApp not found
	at play.mvc.ActionInvoker.getActionMethod(ActionInvoker.java:577)
	at play.mvc.ActionInvoker.resolve(ActionInvoker.java:84)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.Exception: No method public static void listBuyerAccountsApp() was found in class controllers.UserAuthentication
	... 3 more
[2016-08-09 16:50:23,172] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@712oi2c68
UserAuthentication.listBuyerAccountsApp action not found

Action not found
Action UserAuthentication.listBuyerAccountsApp could not be found. Error raised is No method public static void listBuyerAccountsApp() was found in class controllers.UserAuthentication

play.exceptions.ActionNotFoundException: Action UserAuthentication.listBuyerAccountsApp not found
	at play.mvc.ActionInvoker.getActionMethod(ActionInvoker.java:577)
	at play.mvc.ActionInvoker.resolve(ActionInvoker.java:84)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.Exception: No method public static void listBuyerAccountsApp() was found in class controllers.UserAuthentication
	... 3 more
[2016-08-09 16:52:44,193] ERROR [jobs-thread-3] play.Logger.niceThrowable(Logger.java:604) - 

@712oi2c69
Error during job execution (jobs.TaskKeeper)

Compilation error (In /app/controllers/UserAuthentication.java around line 208)
The file /app/controllers/UserAuthentication.java could not be compiled. Error raised is : BuyerAccount cannot be resolved

play.exceptions.CompilationException: BuyerAccount cannot be resolved
	at play.classloading.ApplicationCompiler$2.acceptResult(ApplicationCompiler.java:256)
	at org.eclipse.jdt.internal.compiler.Compiler.handleInternalException(Compiler.java:678)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:522)
	at play.classloading.ApplicationCompiler.compile(ApplicationCompiler.java:292)
	at play.classloading.ApplicationClasses$ApplicationClass.compile(ApplicationClasses.java:289)
	at play.classloading.ApplicationClassloader.detectChanges(ApplicationClassloader.java:326)
	at play.Play.detectChanges(Play.java:636)
	at play.Invoker$Invocation.init(Invoker.java:199)
	at play.jobs.Job.call(Job.java:185)
	at Invocation.Job(Play!)
[2016-08-09 16:53:08,067] INFO  [play-thread-1] jobs.Bootstrap.doJob(Bootstrap.java:20) - =====App Started=====
[2016-08-09 16:56:55,734] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@712oi2c6b
UserAuthentication.doAppLogin action not found

Action not found
Action UserAuthentication.doAppLogin could not be found. Error raised is No method public static void doAppLogin() was found in class controllers.UserAuthentication

play.exceptions.ActionNotFoundException: Action UserAuthentication.doAppLogin not found
	at play.mvc.ActionInvoker.getActionMethod(ActionInvoker.java:577)
	at play.mvc.ActionInvoker.resolve(ActionInvoker.java:84)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.Exception: No method public static void doAppLogin() was found in class controllers.UserAuthentication
	... 3 more
[2016-08-09 16:57:02,119] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@712oi2c6c
UserAuthentication.doAppLogin action not found

Action not found
Action UserAuthentication.doAppLogin could not be found. Error raised is No method public static void doAppLogin() was found in class controllers.UserAuthentication

play.exceptions.ActionNotFoundException: Action UserAuthentication.doAppLogin not found
	at play.mvc.ActionInvoker.getActionMethod(ActionInvoker.java:577)
	at play.mvc.ActionInvoker.resolve(ActionInvoker.java:84)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.Exception: No method public static void doAppLogin() was found in class controllers.UserAuthentication
	... 3 more
[2016-08-09 16:57:26,842] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@712oi2c6d
UserAuthentication.doAppLogin action not found

Action not found
Action UserAuthentication.doAppLogin could not be found. Error raised is No method public static void doAppLogin() was found in class controllers.UserAuthentication

play.exceptions.ActionNotFoundException: Action UserAuthentication.doAppLogin not found
	at play.mvc.ActionInvoker.getActionMethod(ActionInvoker.java:577)
	at play.mvc.ActionInvoker.resolve(ActionInvoker.java:84)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.Exception: No method public static void doAppLogin() was found in class controllers.UserAuthentication
	... 3 more
[2016-08-09 17:00:34,099] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@712oi2c6e
UserCenterApp.doLoginApp action not found

Action not found
Action UserCenterApp.doLoginApp could not be found. Error raised is No method public static void doLoginApp() was found in class controllers.UserCenterApp

play.exceptions.ActionNotFoundException: Action UserCenterApp.doLoginApp not found
	at play.mvc.ActionInvoker.getActionMethod(ActionInvoker.java:577)
	at play.mvc.ActionInvoker.resolve(ActionInvoker.java:84)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.Exception: No method public static void doLoginApp() was found in class controllers.UserCenterApp
	... 3 more
[2016-08-09 17:01:09,825] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@712oi2c6f
UserCenterApp.dologinApp action not found

Action not found
Action UserCenterApp.dologinApp could not be found. Error raised is No method public static void dologinApp() was found in class controllers.UserCenterApp

play.exceptions.ActionNotFoundException: Action UserCenterApp.dologinApp not found
	at play.mvc.ActionInvoker.getActionMethod(ActionInvoker.java:577)
	at play.mvc.ActionInvoker.resolve(ActionInvoker.java:84)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.Exception: No method public static void dologinApp() was found in class controllers.UserCenterApp
	... 3 more
[2016-08-09 17:03:07,328] INFO  [jobs-thread-5] jobs.AddKwdForFlow.doJob(AddKwdForFlow.java:38) - =====AddFlow Started=====
[2016-08-09 17:03:07,336] INFO  [jobs-thread-5] jobs.AddKwdForFlow.doJob(AddKwdForFlow.java:40) - 流量任务条数0
[2016-08-09 17:03:09,456] INFO  [jobs-thread-6] jobs.BuyerTaskCancel.doJob(BuyerTaskCancel.java:38) - cancel buyerTask start
[2016-08-09 17:03:09,469] INFO  [jobs-thread-6] jobs.BuyerTaskCancel.doJob(BuyerTaskCancel.java:49) - cancel buyerTask end
[2016-08-09 17:03:09,480] INFO  [jobs-thread-8] jobs.GetExpressNo.doJob(GetExpressNo.java:49) - =====GetExpressNo Started=====
[2016-08-09 17:03:11,559] INFO  [jobs-thread-8] com.aton.util.FaBaoGuoUtil.execute(FaBaoGuoUtil.java:63) - execute is faild!!
[2016-08-09 17:03:11,564] INFO  [jobs-thread-8] jobs.GetExpressNo.doJob(GetExpressNo.java:59) - fa bao guo return null!
[2016-08-09 17:13:07,374] INFO  [jobs-thread-3] jobs.AddKwdForFlow.doJob(AddKwdForFlow.java:38) - =====AddFlow Started=====
[2016-08-09 17:13:07,384] INFO  [jobs-thread-3] jobs.AddKwdForFlow.doJob(AddKwdForFlow.java:40) - 流量任务条数0
[2016-08-09 17:13:09,527] INFO  [jobs-thread-3] jobs.BuyerTaskCancel.doJob(BuyerTaskCancel.java:38) - cancel buyerTask start
[2016-08-09 17:13:09,533] INFO  [jobs-thread-3] jobs.BuyerTaskCancel.doJob(BuyerTaskCancel.java:49) - cancel buyerTask end
[2016-08-09 17:13:11,638] INFO  [jobs-thread-7] jobs.GetExpressNo.doJob(GetExpressNo.java:49) - =====GetExpressNo Started=====
[2016-08-09 17:13:13,664] INFO  [jobs-thread-7] com.aton.util.FaBaoGuoUtil.execute(FaBaoGuoUtil.java:63) - execute is faild!!
[2016-08-09 17:13:13,664] INFO  [jobs-thread-7] jobs.GetExpressNo.doJob(GetExpressNo.java:59) - fa bao guo return null!
